tasks/update-component-sbom/update-component-sbom.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       description: The workspace where the SBOM files reside.
   steps:
     - name: update-component-sbom-purls
-      image: quay.io/konflux-ci/release-service-utils:221d71a4f6b1a50b36b685aa20d86d7df9de33fc
+      image: quay.io/konflux-ci/release-service-utils:e15023773e138c7d3449f288c8c16de4e3a8d249
```

Comment:
Please update the image used in the test as well - just a convention we have to keep all image refs related to a task in sync.

Reply:
Done

Reply:
Thanks